web/src/app/registration/registration.component.scss

```diff
@@ -7,6 +7,42 @@ form, #formContainer {
     align-items: center;
     font-family: $defaultFont;
     label{
+        align-self:flex-start;
         font-weight: bold;
     }
+}
```

Comment:
The enclosing bracket of `form` /`formContainer` should be moved to the end of the file, since all the elements that come after it are still children of both of these elements.